README.md

```diff
@@ -108,15 +121,22 @@ If you need help with these solutions, please go to the support channel of the Q
-- The QCXR forks we use are: **[Pojlib](https://github.com/questcraftplusplus/pojlib)**, **[VivecraftMod](https://github.com/questcraftplusplus/vivecraftmod)**, and **[VulkanMod](https://github.com/QuestCraftPlusPlus/VulkanMod).** 
-- All support questions should be asked inside of the **[QuestCraft Discord](discord.gg/questcraft)** for the best experience.
-- **questcraft.net is not maintained or owned by anyone on the QuestCraft team**; they could put malware on it anytime. Please do not download anything from or visit questcraft.net. **Our new website is here at [QuestCraft](https://questcraft.org/)**. 
-- Pico (and ONLY Pico) support is planned and is currently in development. Until then, kindly use **[Vivecraft PCVR](https://www.vivecraft.org/)**.
-- When installing mods that are not from Mod Manager on your Quest, you may face issues, ranging from unexpected behaviour to crashes! If you experience any, please **[file an issue](https://github.com/QuestCraftPlusPlus/QuestCraft/issues/new/choose)** or go to the **[Discord server](https://discord.gg/questcraft/)**!
-- Shaders can be used, but expect slideshow frames!
+- The QCXR forks we use are: **[Pojlib](https://github.com/QuestCraftplusplus/pojlib)**, **[VivecraftMod](https://github.com/QuestCraftplusplus/vivecraftmod)**, and **[VulkanMod](https://github.com/QuestCraftPlusPlus/VulkanMod).** 
+- All support questions should be asked inside of the **[QuestCraft Discord](discord.gg/QuestCraft)** for the best experience.
+- **QuestCraft.net is not maintained or owned by anyone on the QuestCraft team**; they could put malware on it anytime. Please do not download anything from or visit QuestCraft.net. **Our new website is here at [QuestCraft](https://QuestCraft.org/)**. 
+- Pico support is planned and is currently in development. Until then, kindly use **[Vivecraft PCVR](https://www.vivecraft.org/)**. 
+- When installing mods that are not from Mod Manager on your Quest, you may face issues, ranging from unexpected behaviour to crashes! If you experience any, please **[file an issue](https://github.com/QuestCraftPlusPlus/QuestCraft/issues/new/choose)** or go to the **[Discord server](https://discord.gg/QuestCraft/)**!
 
 ## Controls
 
 ![There's supposed to be controls here](/Control.png)
 
 These controls are subject to change.
+
+## README Contributors:
+
+<a href="https://github.com/QuestCraftPlusPlus/QuestCraft/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=QuestCraftPlusPlus/QuestCraft" />
+</a>
+
+Made with [contrib.rocks](https://contrib.rocks).
```

Comment:
btw u may want to remove this line

``Made with [contrib.rocks](https://contrib.rocks).``